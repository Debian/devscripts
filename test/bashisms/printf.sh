#!/bin/sh

printf -v some_var "this is a BASHISM"

printf "the use of %q is bad\n" "BASHISMS" >/dev/null

printf "this is another BASHISM: %b" "\n" >/dev/null
