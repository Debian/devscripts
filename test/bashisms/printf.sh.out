possible bashism in bashisms/printf.sh line 3 ('printf -v var ...' should be var='$(printf ...)'):
printf -v some_var "this is a BASHISM"
possible bashism in bashisms/printf.sh line 5 (printf %q|%b):
printf "the use of %q is bad\n" "BASHISMS" >/dev/null
possible bashism in bashisms/printf.sh line 7 (printf %q|%b):
printf "this is another BASHISM: %b" "\n" >/dev/null
